### PR TITLE
LPS-178997 | Poshi Automation for Bug LPS-127451 | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Poshi Automation for Bug LPS-127451

Hi! :)
This pr is corresponding to this ticket: [LPS-127451](https://issues.liferay.com/browse/LPS-127451) → [LPS-178997](https://issues.liferay.com/browse/LPS-178997)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -6646,6 +6646,44 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-127451. Expired web content work in Content Dashboard"
+	@priority = 4
+	test ViewExpiredWebcontentInContentDashboard {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:6651]: Duplicated case with ContentDashboard.ExpiredWebContent

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**